src/import.c

```diff
@@ -35,6 +35,8 @@
  */
 
 gboolean CacheImport(const char *shadow_path, const char *storage_path, const char *username, GError **error) {
+  gboolean result = TRUE;
```

Comment:
Be *very* careful setting default "OK" results. Usually, you want to say that something succeeded as late as possible and leave it fail by default. Your code is bug free right now, it's just a good habit to get into. I'd set it false here then do `result = CacheStoragePutEntry(storage, username, entry, error)` below instead of the if statement.